index.html

```diff
@@ -104,5 +105,199 @@
         </div>
     </div>
 
+    <!-- Content -->
```

Comment:
Chị để ý thằng này vẫn bị lệch. Em dùng div cho thằng to, before cho thằng chấm con nhá. Dùng cả 2 before, after này sẽ không kiểm soát được.
<img width="134" alt="image" src="https://github.com/user-attachments/assets/1730f607-7cbd-4131-a8fb-4a3f21f162d6">